proto/warden/act/v1beta1/tx.proto

```diff
@@ -106,3 +106,13 @@ message MsgRevokeAction {
 }
 
 message MsgRevokeActionResponse {}
+
```

Comment:
please add the new method to the protobuf service as well, see line 26 for example :)

Reply:
Thank you, missed that :(